src/main/paradox/docs/additional-info/benchmarks/deployment-configuration.md

```diff
@@ -3,10 +3,10 @@
 The tests were performed against Nexus services deployed in Kubernetes cluster provisioned by [Amazon EKS](https://aws.amazon.com/eks/).
 The deployment configuration and the number of nodes assigned to each Nexus service are presented in the following diagram:
 
-![deployment configuration](../../assets/img/performance_tests_environment.png)
+![deployment configuration](https://github.com/BlueBrain/nexus/blob/master/src/main/paradox/assets/img/performance_tests_environment.png)
```

Comment:
Can you keep it relative? It will then still work even if we change the domain. Something like 
```
![deployment configuration](/assets/img/performance_tests_environment.png)
```
or
```
![deployment configuration](../../../assets/img/performance_tests_environment.png)
```